nwb_conversion_tools/NWBConverter.py

```diff
@@ -0,0 +1,58 @@
+"""Authors: Cody Baker and Ben Dichter."""
+from copy import deepcopy
+from .utils import get_schema_from_hdmf_class, get_root_schema
+from pynwb import NWBHDF5IO, NWBFile
+from pynwb.file import Subject
+from datetime import datetime
+import uuid
+
+
+class NWBConverter:
+    data_interface_classes = None  # dict of all data interfaces
+
+    @classmethod
+    def get_input_schema(cls):
+        """Compile input schemas from each of the data interface classes."""
+        input_schema = deepcopy(get_root_schema())
```

Comment:
if deepcopy is necessary, it should be moved inside `get_root_schema`

Reply:
Fixed.